src/routes/getSkipSegments.ts

```diff
@@ -79,10 +78,10 @@ function getWeightedRandomChoice(choices, amountOfChoices) {
         //The 3 makes -2 the minimum votes before being ignored completely
         //https://www.desmos.com/calculator/c1duhfrmts
         //this can be changed if this system increases in popularity.
-        const weight = Math.exp((choice.votes + 3), 0.85);
+        const weight = Math.exp((choice.votes + 3));  // TODO TYPESCRIPT what was this 0.85?
```

Comment:
I think it's a mistake, the desmos link suggests it should be y=(x+3)^0.85
I think it should be Math.pow: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow

Unless the algorithm was changed and the comment wasn't updated, though Math.exp should only have one param: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/exp

Reply:
Good catch. Exponent is the opposite of what it's trying to do. It should be `Math.pow()`

Reply:
There are so few votes, so I think leaving it with exponential might be a good idea

Reply:
I added the comment there because IDE started complaining about wrong number of arguments. Didn't notice the comment. Will update that function to use `Math.pow`

Reply:
You can leave this at Math.exp as it has been working well since the change